vendor/github.com/pkg/errors/README.md

```diff
@@ -14,7 +14,7 @@ which applied recursively up the call stack results in error reports without con
 
 ## Adding context to an error
 
-The errors.Wrap function returns a new error that adds context to the original error. For example
+The `errors.Wrap` function returns a new error that adds context to the original error. For example
```

Comment:
pls do not change vendor file directly, if there is issue, pls change dependent project first.

Reply:
Okay I will follow this